src/Plugin/DataCollector.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Prooph\Bundle\ServiceBus\Plugin;
+
+use Exception;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector as BaseDataCollector;
+
+class DataCollector extends BaseDataCollector
+{
+    /** @var string[] */
+    private $busNames = [];
+
+    /** @var ContainerInterface */
+    private $container;
+
+    public function __construct(ContainerInterface $container, string $busType)
+    {
+        $this->container = $container;
+        $this->data['bus_type'] = $busType;
+        $this->data['messages'] = [];
+        $this->data['duration'] = [];
+    }
+
+    public function collect(Request $request, Response $response, Exception $exception = null): void
+    {
+        foreach ($this->busNames as $busName) {
+            $this->data['config'][$busName] = $this->container->getParameter(
+                sprintf('prooph_service_bus.%s.configuration', $busName)
+            );
+        }
+    }
```

Comment:
> User Deprecated: Implementing "**Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface**" 
without the "**reset()**" method is deprecated since version 3.4 and will be unsupported in 4.0 for class "**Prooph\Bundle\ServiceBus\Plugin\DataCollectorPlugin**".

Add it please

Reply:
thank you 👍